streaming/src/test/scala/org/apache/spark/streaming/util/WriteAheadLogSuite.scala

```diff
@@ -508,14 +509,17 @@ class BatchedWriteAheadLogSuite extends CommonWriteAheadLogTests(
     blockingWal.allowWrite()
 
     val buffer1 = wrapArrayArrayByte(Array(event1))
-    val buffer2 = wrapArrayArrayByte(Array(event2, event3, event4, event5))
+    val buffer2 = Set(event2, event3, event4, event5)
```

Comment:
This isnt a buffer any more. Maybe not have `buffer2` at all. And then `buffer1` can be either removed or renamed to `buffer`.